packages/admin-ui/src/components/Sidebar/components/SidebarDisclosure.tsx

```diff
@@ -1,43 +1,109 @@
 import type { ReactNode, Ref } from 'react'
 import React, { forwardRef } from 'react'
 import { merge } from '@vtex/admin-ui-util'
+import { tag, useSystem } from '@vtex/admin-ui-react'
 
+import { Center } from '../../Center'
 import type { ButtonProps } from '../../Button'
 import { Button } from '../../Button'
+import { useSidebarContext } from './SidebarContext'
 
 export const SidebarDisclosure = forwardRef(function SidebarDisclosure(
   props: SidebarDisclosureProps,
   ref: Ref<HTMLButtonElement>
 ) {
-  const { icon, selected, label, csx = {}, ...buttonProps } = props
+  const {
+    icon,
+    selected,
+    selectedFallback,
+    label,
+    csx = {},
+    expandable,
+    ...buttonProps
+  } = props
+
+  const state = useSidebarContext()
+
+  const { keyframes } = useSystem()
+
+  const fadeIn = keyframes`
```

Comment:
Is this a function call `keyframes`?

Reply:
yes, this comes from emotion: https://emotion.sh/docs/keyframes